shared/templates/template_OVAL_grub2_bootloader_argument

```diff
@@ -92,7 +92,7 @@
 
   <ind:textfilecontent54_state id="state_grub2_{{{ SANITIZED_ARG_NAME }}}_argument"
   version="1">
-    <ind:subexpression datatype="string" operation="pattern match">^.*{{{ ESCAPED_ARG_NAME_VALUE }}}.*$</ind:subexpression>
+    <ind:subexpression datatype="string" operation="pattern match">^.*(?:^|\s){{{ ESCAPED_ARG_NAME_VALUE }}}(?:$|\s).*$</ind:subexpression>
```

Comment:
`>^.*(?:^|\s({{{ ESCAPED_` it is a bit strange to expect beginning of line in the group.

Suggestion for the regex:
`^(?:.*\s)?{{{ ESCAPED_`

Reply:
Suggestion for end part of the regex:
`(?:\s.*)?$`